e2e/e2e_redirect_test.go

```diff
@@ -1,14 +1,25 @@
-package httpexpect
+package e2e
 
 import (
+	"github.com/gavv/httpexpect/v2"
 	"io/ioutil"
 	"net/http"
 	"net/http/httptest"
 	"testing"
 
+	"github.com/stretchr/testify/assert"
 	"github.com/valyala/fasthttp"
 )
 
+type booleanReporter struct {
+	failed bool
+}
```

Comment:
Maybe rename it to mockReporter? booleanReporter sounds a bit confusing to me. I think it's not a big deal that we have another mockReporter in another package.